rules/windows/defense_evasion_rundll32_no_arguments.toml

```diff
@@ -22,21 +22,17 @@ tags = ["Elastic", "Windows"]
 type = "eql"
 
 query = '''
-/* replace winlog.event_data.OriginalFileName with process.pe.original_file_name once field is available in winlogbeat */
-
 sequence with maxspan=1h
   [process where event.type in ("start", "process_started") and
-     (process.name == "rundll32.exe" or process.pe.original_file_name == "rundll32.exe" or
-        winlog.event_data.OriginalFileName == "rundll32.exe") and
+     (process.name == "rundll32.exe" or process.pe.original_file_name == "RUNDLL32.EXE") and
 
      /* zero arguments excluding the binary itself (and accounting for when the binary may not be logged in args) */
      ((process.args == "rundll32.exe" and process.args_count == 1) or
       (process.args != "rundll32.exe" and process.args_count == 0))
```

Comment:
`process.args_count` will be 1 at a minimum.
like argv in c/python, the first argument is (typically) the path for rundll32.

Reply:
and if args_count _was_ zero, this `!=` would always be true

Reply:
I have seen the that with certain binaries, the binary can sometimes not be captured as part of the command line, which was the justification for this. I will convert it to normal (your feedback) and if we see the behavior again, revisit this logic